pr-checks/checks/resolve-environment-action.yml

```
@@ -0,0 +1,17 @@
+name: "resolve-environment-action"
```

Minor, for consistency:

```suggestion
name: "Resolve environment"
```